apps/web/app/components/ui/JobCard.tsx

```diff
@@ -16,10 +16,38 @@ const JobCard = ({ job, skillsFromProps }) => {
     return ''
   }, [job.salary])
 
+  const shouldRedirectToUrl = (description: string) => {
+    if (!description) return true
+    const wordCount = description.trim().split(/\s+/).length
+    return wordCount < 10
+  }
```

Comment:
As a rule of thumb, it's not a good idea to create functions inside React components, as they'll be recreated at every new re-render.

Please only create functions inside of react component when it's necessary, such as the `handleClick` one.